onnx/defs/quantization/defs.cc

```diff
@@ -143,15 +143,22 @@ y = saturate (round (x / y_scale) + y_zero_point)
 ```
 )DOC";
 
+static std::vector<int64_t> default_axes = {1};
+
 ONNX_OPERATOR_SET_SCHEMA(
     DynamicQuantizeLinear,
-    11,
+    13,
     OpSchema()
-        .SetDoc(DynamicQuantizeLinear_ver11_doc)
+        .SetDoc(DynamicQuantizeLinear_ver13_doc)
         .Input(0, "x", "Input tensor", "T1")
         .Output(0, "y", "Quantized output tensor", "T2")
         .Output(1, "y_scale", "Output scale. It's a scalar, which means a per-tensor/layer quantization.", "tensor(float)")
         .Output(2, "y_zero_point", "Output zero point. It's a scalar, which means a per-tensor/layer quantization.", "T2")
+        .Attr(
+            "axes",
+            "(Optional) The axis of the quantization dimension of the input tensor. Negative value means counting dimensions from the back. Accepted range is [-r, r-1] where r = rank(input)",
+            AttributeProto::INTS,
```

Comment:
same comment as above. axis here should be same as axis in QuantizeLinear

Reply:
Using axes here was just for experiment. Updated it as axis